src/ui/public/utils/decode_geo_hash.js

```diff
@@ -56,3 +56,17 @@ function refine_interval(interval, cd, mask) {
 }
 
 export default decodeGeoHash;
+
+/**
+ *
+ * @param {number} precision the geohash precision (1<=precision<=12).
+ * @param {number} axis constant for the axis 0=lengthwise (ie. columns, along longitude), 1=heightwise (ie. rows, along latitude)
+ * @returns {number} Number of geohash cells at that precision
+ */
+export function geohashCells(precision, axis) {
+  let cells = 1;
+  for (let i = 1; i <= precision; i += 1) {
+    cells *= (i % 2 === axis) ? 4 : 8;
```

Comment:
I was referring to this [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) definition: "Unique values with unexplained meaning or multiple occurrences which could (preferably) be replaced with named constants".

By putting 4 and 8 into a variable, you're able to give it some context in the code. Right now, I have no idea what this code is doing, and explaining the values may help.